app.js

```diff
@@ -11,6 +11,7 @@ const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
 app.use(logger(formatsLogger))
 app.use(cors())
 app.use(express.json())
+require('./config/config-passport')
 
 app.use('/api/contacts', contactsRouter)
```

Comment:
Tu dodajemy osobne endpointy api dla users
```
app.use('/api/users', contactsRouter)
```
Jak tam już przerzucisz /users do innego pliku to pamiętaj usunąć "/users" z przodu bo tu już to mamy